pkg/lang/python/imports_test.go

```diff
@@ -134,14 +128,20 @@ func TestFindImports(t *testing.T) {
 			source: "from mymodule import attribute1 as a1, attribute2 as a2",
 			want: map[string]Import{
 				"mymodule": {
-					Name:         "mymodule",
-					ImportedSelf: false,
+					Name: "mymodule",
 					ImportedAttributes: map[string]Attribute{
 						"attribute1": {Name: "attribute1", Alias: "a1"},
 						"attribute2": {Name: "attribute2", Alias: "a2"},
 					}},
 			},
 		},
+		{
+			name:   "import sibling",
+			source: "from . import mymodule",
+			want: map[string]Import{
+				".": {ParentModule: ".", ImportedAttributes: map[string]Attribute{"mymodule": {Name: "mymodule"}}},
```

Comment:
Similar to my comment above, but now even more easy to trigger: what happens if someone does:

```python
from . import mymodule
from . import anothermodule
```

Reply:
Should be the same as `import multiple attributes from module in separate imports` but the key of the import is `"."` instead.